Tasks/VSBuild/task.json

```diff
@@ -79,9 +79,10 @@
             "label": "Visual Studio Version",
             "required": false,
             "helpMarkDown": "If the preferred version cannot be found, the latest version found will be used instead.",
-            "defaultValue": "14.0",
+            "defaultValue": "15.0",
             "options": {
                 "latest": "Latest",
+                "15.0": "Visual Studio 15",
```

Comment:
I don't mind either way. I went with "Visual Studio 15" because that's how the branding reads - e.g. splash screen and msdn subscriber download link